ulwgl_run.py

```diff
@@ -310,3 +318,8 @@ def main() -> int:  # noqa: D103
     except Exception as e:  # noqa: BLE001
         print_exception(e)
         sys.exit(1)
+    finally:
+        # Cleanup .ref file on every exit
+        file: Path = Path.home().joinpath(".local/share/ULWGL").joinpath(".ref")
```

Comment:
If I might chip in here, I am very strongly against this kind of overloaded operators. Some operators have very clearly defined usage in other cases, like division here in the case, which makes overloading them essentially code smell.

Reply:
I can respect that it was agreed to not use it

Reply:
It didn't come up until now, so there isn't a previous agreement, tbh. I am just stating my personal view on the matter. Imho, if the operator was something more semantically close to the operation it would be fine. @R1kaB3rN is the one mostly developing and maintaining the python code, so the final decision is theirs.

Reply:
I want the codebase to be as readable as possible. While its a bit more succinct, the result is the same, and sorta reads as traversing a directory from the shell (e.g.` cd ~/.local/share/ULWGL)`, compared to other methods, I believe it can lead to more confusion because the operator conventionally signals a division operation

Unless the efficiency gains, if there is any, outweighs those risks, I can't approve of it

For joining path segments, I believe solutions the involve at least one call to .joinpath()  should be preferred.
Examples:

> Path.home().joinpath(".local/share/ULWGL") # Less preferred because of separators

> Path.home().joinpath(".local", "share", "ULWGL") # Preferred because no separators

>  Path.home().joinpath(".local/" + "share/" + "ULWGL") # Discouraged


Currently the codebase should have the less preferred solutions so I'll accept PRs to change them